frontend/components/FieldImage/FieldImage.jsx

```diff
@@ -9,27 +10,54 @@ import Label from 'components/Label/Label'
 
 export default class FieldImage extends Component { 
 
+  static propTypes = {
+
+    maxWidth: proptypes.number,
```

Comment:
@eduardoboucas it's temporary. I'll make it static for now so that later it can be handled with css